client/state/sites/schema.js

```diff
@@ -57,7 +57,8 @@ export const sitesSchema = {
 						is_free: { type: 'boolean' }
 					}
 				},
-				single_user_site: { type: 'boolean' }
+				single_user_site: { type: 'boolean' },
+				updates: { type: 'object' },
```

Comment:
@gwwar It depends on the type of site really:

* ✅  for single-site Jetpack sites
* ✅  for the main site of Jetpack multisite networks
* ❌  for sub-sites of Jetpack multisite networks
* ❌  for WordPress.com sites

Which is why it is not a required attribute. But we should be adding it to the sites schema, because it's already used here and there (for example, see https://github.com/Automattic/wp-calypso/blob/master/client/lib/sites-list/list.js#L578 or https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/site-indicator/site-indicator.jsx#L30).